scripts/deploy-wavefront.sh

```diff
@@ -19,8 +19,10 @@ WAVEFRONT_PREFIX=$(yq r $PARAMS_YAML wavefront.cluster-name-prefix)
 kubectl create namespace wavefront
 helm repo add wavefront https://wavefronthq.github.io/helm/
 helm repo update
-helm upgrade --install wavefront wavefront/wavefront -f wavefront/wf.yml \
```

Comment:
Are we removing this because most values were defaults?

Reply:
@jaimegag Yes.  Primerally, I was concerned that we have bit-rot with container versions.  It looks like the original file took a complete values.yaml from the helm chart and started making modifications.  My preference for helm is to limit the values.yaml file to only that which is explicitly different than default with explanation of why it is different.  In this case, we are simply wanting to show integration, so keeping it simple is better.  I did want to have @agregory999 have a look as his jaeger integration uses a different values.yaml file for the bonus lab.  I did not modify that at all, but feel that a similar approach should be taken for that.

Reply:
Got it. Makes sense to me. Thanks for the info!